test/type_prop.cpp

```diff
@@ -14446,3 +14446,94 @@ TEST(type_prop, scale_shift)
     EXPECT_EQ(scale_shift_func->get_element_type(), element::f64);
     EXPECT_EQ(scale_shift_func->get_shape(), (Shape{3, 6}));
 }
+
+TEST(type_prop, DISABLED_fake_quantize)
```

Comment:
Lets implement these tests since they are independent of the decomposition for fake_quantize.